modules/standard/CommDiagnostics.chpl

```diff
@@ -372,6 +372,54 @@ module CommDiagnostics
   }
 
 
+  /*
+    Print the current communication counts in a text table showing
+    locales x operations.  By default, operations with a count of
+    zero will not be displayed in the table.
+
+    :arg allOps: Print all operations regardless of count? (defaults to ``false``)
```

Comment:
It wasn't obvious to me what `allOps` meant until I tried running it. I thought it wouldn't print a value if the entry was 0, but I think it's that a column won't be printed if the value is 0 for all locales. Maybe specify that operations whose count on all locales is zero won't be displayed?

Reply:
Think it's worth putting example output in the doc? I could go either way myself.

Reply:
I took another pass on the docs / argument name if you want to take another look.  I was reluctant to put an example in the doc due to maintenance concerns.

Reply:
Looks good to me, thanks